zebra/zebra_vxlan.c

```diff
@@ -5273,10 +5273,6 @@ int zebra_vxlan_process_vrf_vni_cmd(struct zebra_vrf *zvrf, vni_t vni,
 			   add ? "ADD" : "DEL");
 
 	if (add) {
-
```

Comment:
Why is this function moving below the checks, is it b/c it actually may make the checks become true from false?

Reply:
`zebra_vxlan_handle_vni_transition` will change one vni from l2vni to l3vni, it is real things for evpn.
So we must do these checks before real things/`zebra_vxlan_handle_vni_transition`.